openidm-zip/src/main/resources/db/h2/conf/repo.jdbc.json

```diff
@@ -0,0 +1,298 @@
+{
+  "useDataSource" : "h2",
+  "dbType" : "H2",
+  "queries" : {
+        "genericTables" : {
+            "credential-query" : "SELECT fullobject FROM ${_dbSchema}.${_mainTable} obj INNER JOIN ${_dbSchema}.objecttypes objtype ON objtype.id = obj.objecttypes_id AND objtype.objecttype = ${_resource} INNER JOIN ${_dbSchema}.${_propTable} usernameprop ON obj.id = usernameprop.${_mainTable}_id AND usernameprop.propkey='/userName' INNER JOIN ${_dbSchema}.${_propTable} statusprop ON obj.id = statusprop.${_mainTable}_id AND statusprop.propkey='/accountStatus' WHERE usernameprop.propvalue = ${username} AND statusprop.propvalue = 'active'",
+            "get-by-field-value" : "SELECT fullobject FROM ${_dbSchema}.${_mainTable} obj INNER JOIN ${_dbSchema}.${_propTable} prop ON obj.id = prop.${_mainTable}_id INNER JOIN ${_dbSchema}.objecttypes objtype ON objtype.id = obj.objecttypes_id WHERE prop.propkey=CONCAT('/', ${field}) AND prop.propvalue = ${value} AND objtype.objecttype = ${_resource}",
+            "query-all-ids" : "select objectid from ( select /*+ FIRST_ROWS(n) */ a.*, ROWNUM rnum from (SELECT obj.objectid as objectid FROM ${_dbSchema}.${_mainTable} obj INNER JOIN ${_dbSchema}.objecttypes objtype ON obj.objecttypes_id = objtype.id WHERE objtype.objecttype = ${_resource} order by obj.id ) a where ROWNUM <= ${int:_pagedResultsOffset}+${int:_pageSize}) where rnum > ${int:_pagedResultsOffset}",
```

Comment:
The same question as above... I can see that the same thing is inside Oracle's repo configuration. If H2 is the default test database, the config should be pristine. So please correct those issues and fix the formatting as well (uppercase keywords). This will be the config file all other repo configs are based on.

Reply:
comment removed